BREEZE.rst

```diff
@@ -2106,13 +2106,30 @@ Those are all available flags of ``generate-constraints`` command:
 
 In case someone modifies setup.py, the scheduled CI Tests automatically upgrades and
 pushes changes to the constraint files, however you can also perform test run of this locally using
-the procedure described in `Refreshing CI Cache <dev/REFRESHING_CI_CACHE.md#manually-generating-constraint-files>`_
+the procedure described in the
+`Manually generating image cache and constraints <dev/MANUALLY_GENERATING_IMAGE_CACHE_AND_CONSTRAINTS.md>`_
 which utilises multiple processors on your local machine to generate such constraints faster.
 
 This bumps the constraint files to latest versions and stores hash of setup.py. The generated constraint
 and setup.py hash files are stored in the ``files`` folder and while generating the constraints diff
 of changes vs the previous constraint files is printed.
 
+Updating constraints
+""""""""""""""""""""
+
+Sometimes (very rarely) we might want to update individual packages in constraints that we generated and
+tagged already in the past. This can be done using ``breeze release-management update-constraints`` command.
+
+Those are all available flags of ``update-constraints`` command:
```

Comment:
nit: `These`

Reply:
Well. It was used in (literally) 30 other places in BREEZE instructions, So I went ahead and updated all of those (or these?) I never know.

Reply:
Small nit that i came across while reading the PR. Feel free to ignore it :)

Reply:
Already fixed it :)

Reply:
In all 30 places

Reply:
Thanks!